{{ cookiecutter.project_slug }}/README.md

```diff
@@ -18,15 +18,18 @@ Start the dev server for local development:
 ```bash
 docker-compose up
 ```
+You can access your local environment at [http://localhost:8000](http://localhost:8000)
 
 Enter the docker container (while the server is running), migrate, and create a user for yourself:
 ```bash
 # Enters a shell in the docker container
 docker-compose exec web bash
 
 # And create a superuser for yourself.
+# You'll be prompted to enter a first name, last name, email, password, and password confirmation
 $ ./manage.py createsuperuser
 ```
+After creating the superuser you can access the admin panel at [http://localhost:8000/admin/](http://localhost:8000/admin/)
```

Comment:
I tried using `/admin` instead of `/admin/` had to look at the docs to see that the trailing slash was required.